hyperSpec/R/generate-test-data.R

```diff
@@ -0,0 +1,115 @@
+# These functions generate datasets that are mainly used for unit testing.
+# V. Gegzna
+
+# Generate spectra -----------------------------------------------------------
+# Generate data that contains several spectra
+
+# n_wl - number of wavelengths (points / spectrum).
+# n - number of spectra.
+
+generate_hy_spectra <- function(n_wl = 50, n = 20) {
```

Comment:
@cbeleites 

> Should these functions be exported, so that we can use them for testing in hySpc.*?

I think It would be easier if they were exported.

> Should these functions be hidden?

I don't think that exported objects should be hidden.

> Should the test data be completely deterministic

Is there a necessity for variation in test data? I did it totally deterministic in order to diminish the possibilities of breaking unit tests due to changes in data.

> would be sensible to assign a test data set once into a [...] variable

In some situations, slight variations of datasets are needed (I'm not sure if I included those tests in this particular pull request)

Reply:
The functions were documented and exported in bb52883